### ISSUE TITLE
some stuff (

### ISSUE THREAD

**Issue by tunnckoCore** [jonschlinkert/gh-clone#5]
1) SSH support

2) I think it should exit with 1, when in the `bin/clone.js` when `!repo` ([after the L23](https://github.com/jonschlinkert/gh-clone/blob/master/bin/clone.js#L23))

``` sh
% 1 ❯ gh-clone                                               July 24, 22:19:56

[22:21:01] Please provide a `repo` either as a first argument or with `-r`
[22:21:01] gh-clone [repo]  · undefined ✔

[22:21:01] Cloning.


[22:21:01] expected `repo` to be a string. Example: `owner/name`
~  
% 1 ❯ 
```

Make more sense to exit earlier.

Other than that, it is great and super fast!

**Comment by jonschlinkert** [comment]
Agreed with all, wan to do a pr? or I can look at implementing next time I make changes

**Comment by tunnckoCore** [comment]
Yea, no problem, when I have time too :D

**Comment by laggingreflex** [comment]
I created a very similar project a while back: [gfork](https://github.com/laggingreflex/gfork). It has SSH support (it can clone the SSH URLs), and more! Check it out! :). There were also other similar projects out there (checkout similar projects section) but they all lacked one thing or the other so I wanted to create one that does it all. The features it currently lacks from this project are `--branch` and API usage. Would love to add it in, or PRs welcome.

**Comment by jonschlinkert** [comment]
> I created a very similar project a while back: gfork. It has SSH support (it can clone the SSH URLs), and more!

Nicely done! thanks for the link!
